CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Changelog
 -----
 
+#### 1.1.3
```

Comment:
This is a new feature, so we'll go to 1.2.0 😄

I can fix the README though, no worries.